### PR TITLE
Code Readability - Power

### PR DESCRIPTION
**Pull request by KalevTait**
## What Does This PR Do
* Converts 0/1 to FALSE/TRUE where those #defines make the code more readable.
* Renames `/obj/singularity/proc/dissipate()` to `/obj/singularity/proc/do_dissipate()`, to reduce confusion with the `var/dissipate` inside of  `/obj/singularity`

## Why It's Good For The Game
More readable code is easier to work with.

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -65,27 +65,27 @@
...
 	var/opened = 0 //0=closed, 1=opened, 2=cover removed
-	var/shorted = 0
+	var/shorted = FALSE
 	var/lighting = 3
 	var/equipment = 3
 	var/environ = 3
-	var/operating = 1
-	var/charging = 0
+	var/operating = TRUE
+	var/charging = 0 // 0 is not charging, 1 is charging, 2 is fully charged
```
Comment (farie82) [inline code/modules/power/apc.dm:73]: Could you make this use defines for the states?

Comment (KalevTait) [reply code/modules/power/apc.dm]: I've made opened, charging and electronics_state (renamed from has_electronics) use defines for states.
I have not converted autoflag to use defines for states, as I'm not confident I understand what it is doing when it has a value set outside the range that it's supposed to have...

Comment (KalevTait) [reply code/modules/power/apc.dm]: I'm not sure why my commit got named 'eye_protection ='...

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -64,39 +79,39 @@
...
-	var/shorted = 0
+	var/opened = APC_CLOSED
+	var/shorted = FALSE
 	var/lighting = 3
 	var/equipment = 3
 	var/environ = 3
-	var/operating = 1
-	var/charging = 0
+	var/operating = TRUE
+	var/charging = APC_NOT_CHARGING
```
Comment (farie82) [inline code/modules/power/apc.dm:88]: This field is used in multiple files. You can check this in vscode by right clicking on the field and selecting "Find all references"
image

Comment (KalevTait) [reply code/modules/power/apc.dm:83]: I wish I'd known about checking references like that two weeks ago - I guess that's what I get for not using a proper IDE for years.

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**